Automattic/wp-calypso
# Store: Get coupons via WooCommerce API

This adds the data-layer http code necessary to fetch pages of coupons
from the API.

To Test:
1. `npm test`
2. `npm start`
3. Visit `http://calypso.localhost:3000/store/promotions/<site url>`
4. Open Redux Dev Tools and filter actions by `COUPONS`
5. View the `WOOCOMMERCE_COUPONS_PAGE_UPDATED` action and verify data in the fields matches what you expect from your site.
